tests/unit/test_louisHelper.py

```diff
@@ -0,0 +1,113 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2024 NV Access Limited, Leonard de Ruijter
+
+"""Unit tests for the louisHelper module.
+"""
+
+import os.path
+import unittest
+
+import brailleTables
+import louisHelper
+import NVDAState
+
+
+class TestResolvingInternal(unittest.TestCase):

```

Comment:
can you add test cases for both of these classes to test the case where a table cannot be resolved